docs/en/enterprise-edition/rn/prisma-cloud-release-info/features-introduced-in-2024/features-introduced-in-february-2024.adoc

```diff
@@ -312,6 +473,211 @@ Prisma Cloud will update the `gcloud-compute-autoscaler` API to exclude the `rec
+----
+
+
+|*Azure Machine learning workspace is not configured with private endpoint*
+
+tt:[*24.2.2*]
+
+//RLP-126235
+
+|This policy identifies Azure Machine learning workspaces that are not configured with private endpoint. Private endpoints in workspace resources allow clients on a virtual network to securely access data over Azure Private Link. Configuring a private endpoint enables access to traffic coming from only known networks and prevents access from malicious or unknown IP addresses which includes IP addresses within Azure. It is recommended to create private endpoint for secure communication for your Machine learning workspaces.
+
+*Policy Severity—* Medium
+
+*Policy Type—* Config
+
+----
+config from cloud.resource where cloud.type = 'azure' AND api.name = 'azure-machine-learning-workspace' AND json.rule = properties.provisioningState equal ignore case Succeeded and (properties.privateEndpointConnections[*] does not exist or properties.privateEndpointConnections[*] is empty or (properties.privateEndpointConnections[*] exists and properties.privateEndpointConnections[*].properties.privateLinkServiceConnectionState.status does not equal ignore case Approved))
+----
+
+
+|*Azure Kubernetes Service (AKS) container service that is internet reachable with unrestricted access (0.0.0.0/0)*
+
+tt:[*24.2.2*]
+
+//RLP-126309
+
+|This policy identifies Azure Kubernetes Service (AKS) container services that are internet reachable with unrestricted access (0.0.0.0/0). Container services with unrestricted access to the internet may enable bad actors to use brute force on a system to gain unauthorized access to the entire network. As a best practice, restrict traffic from unknown IP addresses and limit access to known hosts, services, or specific entities.
+
+*Policy Severity—* High
+
+*Policy Type—* NetworkConfig
+
+----
+config from network where source.network = '0.0.0.0/0' and address.match.criteria = 'full_match' and dest.resource.type = 'K8s Service' and dest.cloud.type = 'AZURE' and effective.action = 'Allow'
+----
+
+
+|*AWS EKS K8s service that is internet reachable with unrestricted access (0.0.0.0/0)*
+
+tt:[*24.2.2*]
+
+//RLP-126545
+
+|This policy identifies AWS EKS K8s service that are internet reachable with unrestricted access (0.0.0.0/0). Containers with unrestricted access to the internet may enable bad actors to use brute force on a system to gain unauthorized access to the entire network. As a best practice, restrict traffic from unknown IP addresses and limit access to known hosts, services, or specific entities.
+
+*Policy Severity—* High
+
+*Policy Type—* Network
+
+----
+config from network where source.network = '0.0.0.0/0' and address.match.criteria = 'full_match' and dest.resource.type = 'K8s Service' and dest.cloud.type = 'AWS' and effective.action = 'Allow'
+----
+
+
+|*IBM Cloud Block Storage volume for VPC is not encrypted with BYOK*
```

Comment:
Please check if this policy must be removed since IBM is not yet GA.  You can track the updates here: https://redlock.atlassian.net/browse/RLP-127891?focusedCommentId=956120